engines/build.info

```diff
@@ -42,7 +44,9 @@ IF[{- !$disabled{"engine"} -}]
     IF[{- !$disabled{capieng} -}]
       MODULES{engine}=capi
       SOURCE[capi]=e_capi.c
-      DEPEND[capi]=../libcrypto
+      IF[{- $target =~ m/^linux/ -}]
+        DEPEND[padlock]=../libcrypto
```

Comment:
this looks wrong, both because this section is about capi, and because capi is windows only.

Reply:
because it is an interface to the windows crypto functions declared in <wincrypt.h>:
https://github.com/openssl/openssl/blob/04bc3c1277b8b20dc29f96933f7be592c0535aa8/engines/e_capi.c#L13-L26

Reply:
It is still built on linux, all 15072 bytes of it. If it shouldn't be built on linux, it is probably a separate problem.

Reply:
This patch has absolutely no effect on windows.

Reply:
Still, this line should say capi, not padlock

Reply:
Yep. I fixed the patch. Some comments where `pending` and then lost